Add entry ember europe london (#1924)
mainmatter/mainmatter.com: src/calendar/2022-12-08-embereurope-london.md

---
title: "Ember Europe"
image: "/assets/images/calendar/2022-12-08-embereurope-london/embereurope.png"
location: London & remote
url: https://www.meetup.com/ember-europe/
kind: meetup
color: aqua
---

> The Europe Ember.js meetup is a one-day event, bringing together the Ember
> community from all of Europe once a quarter. The meetup is organized as a hybrid
> event to allow people to join remotely as well.

Reviewer: ```suggestion
Ember Europe series is back in Q4 with the last event for the year, this time hosted by CrowdStrike in London. The Ember Europe meetup series brings together the Ember community from all of Europe once a quarter. The meetup is organized by Mainmatter and Intercom as a hybrid event, hosted from a different city each time with the option for people to join remotely as well.
```

---

> title: "Ember Europe"

Reviewer: Let's also use a better title

```suggestion
title: "Ember Europe Q4 Meetup"
```

image: "/assets/images/calendar/2022-12-08-embereurope-london/embereurope.png"
location: London & remote
url: https://www.meetup.com/ember-europe/
kind: meetup
color: aqua
---

> Ember Europe series is back in Q4 with the last event for the year, this time

Reviewer: ```suggestion
Ember Europe is back in Q4 with the last event for the year, this time
```

If using series I think it requires "The " as well but that's how the next sentence starts

Author: You wrote the text, I just committed your suggestion :D. I'll change it
